modules/installation-creating-aws-bootstrap.adoc

```diff
@@ -40,6 +40,11 @@ Ignition config files for your cluster are served from an S3 bucket. If you
 choose to serve the files from another location, you must modify the templates.
 ====
 +
+[IMPORTANT]
```

Comment:
So I think most users who are using gov cloud or special endpoints already understand that s3:// wouldn't work for them. 

But I think for helping users this addition should be good.